packages/wonder-blocks-button/components/button-core.js

```diff
@@ -158,18 +158,9 @@ const _generateStyles = (color, kind, light) => {
                 color: light ? color : white,
             },
             focus: {
-                ":before": {
-                    content: "''",
-                    position: "absolute",
-                    top: -3,
-                    left: -3,
-                    right: -3,
-                    bottom: -3,
-                    borderColor: light ? white : color,
-                    borderWidth: 2,
-                    borderStyle: "solid",
-                    borderRadius: 7,
-                },
+                boxShadow: `0 0 0 1px ${light ? darkBlue : white}, 0 0 0 3px ${
```

Comment:
Ah - this is a great solution! Very easy to understand. It might be worth just adding a comment here that we're making the assumption that the background will be a particular color.